rustls-platform-verifier/src/android.rs

```diff
@@ -68,41 +87,76 @@ fn global() -> &'static Global {
         .expect("Expect rustls-platform-verifier to be initialized")
 }
 
-/// Initializes and stores the required context for the Android platform.
+/// Initialize given a typical Android NDK `JNIEnv` and `jobject` context.
 ///
-/// This method will setup and store an environment locally. This is useful if
-/// nothing else in your application needs access the Android runtime.
-///
-/// Initialization must be done before any verification is attempted.
-pub fn init_hosted(env: &mut JNIEnv, context: JObject) -> Result<(), JNIError> {
+/// This method will setup and store an environment locally. This is useful if nothing else in your
+/// application needs access the Android runtime.
```

Comment:
This is my typo originally: "needs access the Android runtime", but if you wouldn't mind squashing it in your existing rustdoc commit I'd be appreciated 😄.